.env.example

```
@@ -28,4 +22,21 @@
+
+# Upstash Redis (for AI rate limiting and caching)
+# Get from: https://upstash.com/
+UPSTASH_REDIS_REST_URL="https://your-redis-url.upstash.io"
+UPSTASH_REDIS_REST_TOKEN="your-redis-token-here"
+
+# Weather API (for smart gardening recommendations)
+# Get from: https://www.weatherapi.com/
+WEATHER_API_KEY="your-weather-api-key-here"
+
+# Anthropic API Key
+ANTHROPIC_API_KEY=your-api-key-here  # Format: sk-ant-api03-...
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)